packages/design-system/src/styles/components/_Choice.scss

```diff
@@ -13,23 +13,22 @@ $ds-c-inset-border-width: $spacer-half;
 
   // Choice label
   + label {
-    align-items: center;
+    align-items: flex-start;
+    cursor: pointer;
```

Comment:
As per [this feedback here](https://github.com/CMSgov/design-system/pull/841#issuecomment-705668831), I think we should be applying the cursor to the `span` and the `:before` elements inside a `label`, not to the label itself.

Reply:
Instead of trying to make the label width smaller, we should make the children of the label not expand to the full width of the parent label.

Reply:
Good pick up on the `$measure-base` utility. The issue in using this is that the `clickable area` is the width of the `$measure-base`, but the cursor pointer is only showing for the label text. 

I've changed the cursor pointer/not-allowed to apply only to the label text. 

Do you think we should still apply the cursor pointer/not-allowed only to the label text or is more accurate to have the cursor pointer apply for the `$measure-base` width?